### PR TITLE
feat(dependencies): link to written offer

### PR DESCRIPTION
**Pull request by MaxTru**
* Legal change
* Related to camunda/engineering-leadership-team#68
* Planning this change for the October release

## When should this change go live?

- [x] This is a bug fix, security concern, or something that needs **urgent release support**.
- [x] This is already available but undocumented and should be released within a week.
- [ ] This on a **specific schedule** and the assignee will coordinate a release with the DevEx team. (apply `hold` label or convert to draft PR)
- [ ] This is part of a scheduled alpha or minor. (apply alpha or minor label)
- [ ] There is **no urgency** with this change and can be released at any time.

## PR Checklist

- [ ] My changes are for an **already released minor** and are in `/versioned_docs` directory.
- [x] My changes are for the **next minor** and are in `/docs` directory (aka `/next/`).

- [ ] My changes require an [Engineering review](https://github.com/camunda/camunda-docs/blob/main/howtos/documentation-guidelines.md#review-process), and I've assigned the Engineering DRI or delegate.
- [x] My changes require a [technical writer review](https://github.com/camunda/camunda-docs/blob/main/howtos/documentation-guidelines.md#review-process), and I've assigned @camunda/tech-writers as a reviewer.

### CODE REVIEW

File: docs/reference/dependencies.md [hunk 1]
```diff
@@ -1,11 +1,11 @@
 ---
 id: dependencies
-title: "Dependencies"
+title: "Source-code and third-party dependencies"
 description: "Dependencies and Third Party Libraries for all the components of Camunda 8"
 keywords: ["dependencies", "third party", "third party libraries"]
 ---
 
-A complete list of all dependencies and third-party libraries for all the components of Camunda 8 (including Self-Managed).
+Camunda provides its users and customers access to the source code under [this written offer](link) and a CyclonDX SBOM with a list of all third-party dependencies and their respective licenses included. Please click on the below tab to get access to these for the respective Camunda component.
```
[the remark this answers is not in the document]
Comment (MaxTru) [reply docs/reference/dependencies.md:8]: You are right. Thanks Amara. Will adjust later on

File: docs/reference/dependencies.md [hunk 2]
```diff
@@ -1,90 +1,66 @@
 ---
 id: dependencies
-title: "Dependencies"
+title: "Source-code and third-party dependencies"
```
Comment (mesellings) [inline docs/reference/dependencies.md:3]:
```suggestion
title: "Source code and third-party dependencies"
```
Not sure the hyphen should be here?

**Review by mesellings — APPROVED** [review]
Lgtm - just a few (non-blocking) suggestions 👍

**Comment by MaxTru** [comment]
@mesellings agree with your suggestion, but this was already merged 🙈 

Can you please follow up and do these improvements in a separate PR?

**Comment by mesellings** [comment]
@MaxTru Added these edits into https://github.com/camunda/camunda-docs/pull/4409 👍

**Pull request merged**